content/providers/01-ai-sdk-providers/03-openai.mdx

```diff
@@ -1332,11 +1333,12 @@ const model = openai.image('dall-e-3');
 
 ### Model Capabilities
 
-| Model         | Sizes                           |
-| ------------- | ------------------------------- |
-| `gpt-image-1` | 1024x1024, 1536x1024, 1024x1536 |
-| `dall-e-3`    | 1024x1024, 1792x1024, 1024x1792 |
-| `dall-e-2`    | 256x256, 512x512, 1024x1024     |
+| Model              | Sizes                           |
+| ------------------ | ------------------------------- |
+| `gpt-image-1-mini` | 1024x1024, 1536x1024, 1024x1536 |
```

Comment:
I verified by setting an incorrect size, the error included all permitted sizes which were these three